### PR TITLE
Issue #10157: Updated inputs for EmptyCatchBlockCheckTest

### LINKED ISSUE #10157: Update inputs for EmptyCatchBlockCheckTest
Child of #10080

### PR DESCRIPTION
**Pull request by shashwatj07**
Resolves #10157

### CODE REVIEW

**Comment by timurt** [comment]
```
[ERROR] [checkstyle] [ERROR] C:\projects\checkstyle\src\test\resources\com\puppycrawl\tools\checkstyle\checks\blocks\emptycatchblock\InputEmptyCatchBlockDefaultLF.java:0: All check input files in the 'src/(it|test)/resources*/' folder must                    have trailing comments "// ok" and/or "// violation". [StyleValidationCommentInInputs]
```

**Comment by shashwatj07** [comment]
> ```
> [ERROR] [checkstyle] [ERROR] C:\projects\checkstyle\src\test\resources\com\puppycrawl\tools\checkstyle\checks\blocks\emptycatchblock\InputEmptyCatchBlockDefaultLF.java:0: All check input files in the 'src/(it|test)/resources*/' folder must                    have trailing comments "// ok" and/or "// violation". [StyleValidationCommentInInputs]
> ```

resolved @timurt

**Comment by timurt** [comment]
```
Actual:
EmptyCatchBlockCheck.java.html:<td class='covered'><pre><span  class='survived'>                if (!line.isEmpty()) {</span></pre></td></tr>
Ignore:

Diff:
--- target/ignored.txt	2021-06-24 11:35:59.477231113 +0000
+++ target/actual.txt	2021-06-24 11:35:59.477231113 +0000
@@ -0,0 +1 @@
+EmptyCatchBlockCheck.java.html:<td class='covered'><pre><span  class='survived'>                if (!line.isEmpty()) {</span></pre></td></tr>
Difference between 'Actual' and 'Ignore' lists is detected, lists should be equal.
build will be failed.
```

**Comment by romani** [comment]
https://91842-12499251-gh.circle-artifacts.com/0/target/pit-reports/202106241134/com.puppycrawl.tools.checkstyle.checks.blocks/EmptyCatchBlockCheck.java.html

if go to artifacts, and fins failed container, there is HTML report from pitest.

File: src/test/resources/com/puppycrawl/tools/checkstyle/checks/blocks/emptycatchblock/InputEmptyCatchBlockDefault2.java [hunk 1]
```diff
@@ -0,0 +1,235 @@
...
+    }
+
+    private void some() {
+        try {
+            throw new IOException();
+        } catch (IOException e) { // violation
+            /* ololo
+             * blalba
+             */
+        }
```
Comment (shashwatj07) [inline src/test/resources/com/puppycrawl/tools/checkstyle/checks/blocks/emptycatchblock/InputEmptyCatchBlockDefault2.java:181]: We have a slight problem here. Mutation survives on adding ```// violation``` comment. How to handle these edge cases?

Comment (pbludov) [reply src/test/resources/com/puppycrawl/tools/checkstyle/checks/blocks/emptycatchblock/InputEmptyCatchBlockDefault2.java:181]: Just a guess,
```java
        } catch (IOException e) { /* violation
             * blalba
             */
        }
```
?

Comment (shashwatj07) [reply src/test/resources/com/puppycrawl/tools/checkstyle/checks/blocks/emptycatchblock/InputEmptyCatchBlockDefault2.java:181]: Yes, I thought of the same thing. We'll then have to modify our regexp in the parser. Will that be fine?

Comment (romani) [reply src/test/resources/com/puppycrawl/tools/checkstyle/checks/blocks/emptycatchblock/InputEmptyCatchBlockDefault2.java:181]: What about `} catch (IOException e) { /* //violation`
?

Comment (shashwatj07) [reply src/test/resources/com/puppycrawl/tools/checkstyle/checks/blocks/emptycatchblock/InputEmptyCatchBlockDefault2.java]: done

**Pull request merged**